hivemq/assets/dashboards/hivemq.json

```diff
@@ -2,24 +2,30 @@
     "author_info": {
         "author_name": "Datadog"
     },
-    "board_title": "HiveMQ Overview",
+    "board_title": "HiveMQ",
```

Comment:
I'm not sure if we can update the board_title once it's synced. The url path is dependent on this, for example the default dash right now is https://app.datadoghq.com/screen/integration/30331/hivemq-overview

Reply:
Did not know that. I can change it back to HiveMQ Overview.

Reply:
Actually that does not seem to be a problem at all, only the id seems to matter:

Example: [https://app.datadoghq.com/screen/integration/30331/RANDOM_NAME_THAT_DOESNT_MATTER](https://app.datadoghq.com/screen/integration/30331/RANDOM_NAME_THAT_DOESNT_MATTER) still points to the HiveMQ dashboard

Reply:
It looks like the board_title does have an impact, going to make a PR to change it to its original value of `HiveMQ Overview`